conda_forge_tick/migrate_update_version.py

```diff
@@ -0,0 +1,43 @@
+# type this code in at the interpreter
+from conda_forge_tick.utils import load_graph
+from conda_forge_tick.update_upstream_versions import get_latest_version, PyPI, CRAN, NPM, ROSDistro, RawURL, Github
+import random
+import json
+sources = (PyPI(), CRAN(), NPM(), ROSDistro(), RawURL(), Github())
+
+print("Reading graph")
+gx = load_graph()
+_all_nodes = [t for t in gx.nodes.items()]
+random.shuffle(_all_nodes)
+
+# from here you can inspect the graph object, and what node will be updated:
+print("Number of nodes:", len(gx.nodes))
+Node_count = 0
+
+to_update = {}
+to_update["nodes"] = []
+for node in gx.nodes:
+    with gx.nodes["{}".format(node)]["payload"] as attrs:
+        # verify the actual situation of the package;
+        if attrs.get("bad") or attrs.get("archived"):
+            new_version_flag = False
+        
+        # new version request
+        try:
+            new_version = get_latest_version(node, attrs, sources)
```

Comment:
I wouldn't try to get the latest version if the feedstock has been marked bad/archived

Reply:
ok, i will put a flag for that, thanks.

Reply:
Done,